modules/host/storageobligations.go

```diff
@@ -869,15 +869,38 @@ func (h *Host) StorageObligations() (sos []modules.StorageObligation) {
 			if err != nil {
 				return build.ExtendErr("unable to unmarshal storage obligation:", err)
 			}
+			statusString := map[storageObligationStatus]string{
```

Comment:
Can we achieve this in a cleaner way by adding a `String()` method to the `storageObligationStatus` type?

Reply:
I considered that too. I think it would be an improvement, although not a huge one, because `storageObligationStatus` isn't exported.

Reply:
btw, this is what `stringer` was created to do: https://godoc.org/golang.org/x/tools/cmd/stringer